src/ui/Facet/FacetValuesList.ts

```diff
@@ -119,11 +118,7 @@ export class FacetValuesList {
   }
 
   protected getValuesToBuildWith() {
-    if (this.facet.facetSort) {
-      return new FacetValuesOrder(this.facet, this.facet.facetSort).reorderValues(this.facet.values.getAll());
-    } else {
-      return this.facet.values.getAll();
-    }
+    return this.facet.values.getAll();
```

Comment:
The second half of the key to this PR, avoiding an alphabetical sort at this stage because the values have been sorted by status, and we do not want to lose that information.